packages/tldraw/src/lib/TldrawImage.tsx

```diff
@@ -184,6 +167,12 @@ export const TldrawImage = memo(function TldrawImage(props: TldrawImageProps) {
 		pixelRatio,
 	])
 
+	useLayoutEffect(() => {
```

Comment:
i could be wrong but i don't think this needs to be `useLayoutEffect` (i.e. synchronous)

i think in general, for the places we use `useLayoutEffect` we should have a justification comment for it: "We're using layoutEffect here b/c x,y,z..."

Reply:
ah yeah - i wasn't sure if a standard effect would always run if a layout effect caused several syncronous renders, but looks like it does!